src/components/tooltip/index.css

```diff
@@ -0,0 +1,21 @@
+.hx-tooltip {
+    box-shadow: none;
+    border: none;
+    overflow: visible;
+}
+
+.hx-tooltip-label  {
+    border-bottom: 1px dotted transparent;
+}
+
+.hx-tooltip-icon,
+.hx-tooltip-label {
+    cursor: pointer;
+}
+
+.hx-tooltip:before {
```

Comment:
Need to add support for when the tooltip is close to the top of the page (and the dropdown is shown below instead of above)